build-tools/packages/build-cli/src/library/package.ts

```diff
@@ -68,9 +68,7 @@ export async function npmCheckUpdates(
 	depsToUpdate: ReleasePackage[] | RegExp[],
 	releaseGroupFilter: ReleaseGroup | undefined,
 	depUpdateType: DependencyUpdateType,
-	// eslint-disable-next-line default-param-last
```

Comment:
Was this rule removed?

Reply:
Not with this update. Those disables were unused prior to this update. The removals were applied by running lint with --fix.
The rule is not so relevant to TypeScript anyway - tsc will complain if there are required parameters without defaults after optional/defaulted parameters.